vtds_cluster_kvm/private/scripts/kickstart.py

```diff
@@ -0,0 +1,740 @@
+#! python
```

Comment:
REVIEWERS: this class builds a Kicstart configuration file from the cluster configuration during cluster deployment. This file is one of the python libraries sent to the blade during the `prepare` phase.